### PR TITLE
API-11368 UI style

### PR DESCRIPTION
**Pull request by crolarlibertyva**
- When in "refactored UI" mode, UI is similar in style to staging.va.gov
- When not in "refactored UI" mode, UI style is unchanged
Screen Shot 2021-12-03 at 2 11 19 PM

### CODE REVIEW

File: views/login_selection_refactor.hbs [hunk 1]
```diff
@@ -80,11 +53,13 @@
               </p>
               <p>
                 If you need more help, call our MyVA411 main information line at
```
Comment (dbrown-va) [inline views/login_selection_refactor.hbs:55]: `If you need more help, ` looks to be removed from the VA.gov page.

image

File: views/login_selection_refactor.hbs [hunk 2]
```diff
@@ -1,68 +1,41 @@
...
-              <div class="alternate-signin">
-                <h2 class="vads-u-font-size--sm vads-u-margin-top--0">
-                  Don't have those accounts?
-                </h2>
-                {{#if login_gov_signup_link_enabled}}
-                  <a class="no-external-icon usa-button signup" href="{{ login_gov_signup_link }}"><img alt="Login.gov" src="/samlproxy/idp/img/login-gov-dark-logo.svg" />Create a Login.gov account</a>
-                {{/if}}
-                <a class="no-external-icon usa-button signup" href="{{ id_me_signup_link }}"><img alt="ID.me" src="/samlproxy/idp/img/idme-icon-dark.svg" />Create an ID.me account</a>
-              </div>
+              <a class="vads-c-action-link--blue" href="{{ id_me_signup_link }}">Create an accoutn with ID.me</a>
```
Comment (dbrown-va) [inline views/login_selection_refactor.hbs:28]: Spelling issue here. `accoutn`

image

**Comment by beaugrantham-va** [comment]
- [ ] button spacing is larger than VA.gov
- [ ] login.gov button font is different than login.gov
- [ ] "signing in" link points to staging VA.gov
- [ ] VA.gov uses hover pseudo classes to change button color

**Comment by dbrown-va** [comment]
UI is very close to matching VA.gov for desktop viewing.  Button SVG's and spacing all looking good with the latest updates.
image


However when I switched to mobile rendering, the styling runs into issues.

image


Versus staging.va.gov

image

**Review by dbrown-va — DISMISSED** [review]
Mobile rendering is running into issues when the UI refactor is enabled.

I think related to css classes being applied based on pixel width, because shrinking the width of desktop browser produced similar diverging behavior.

**Review by beaugrantham-va — DISMISSED** [review]
Something is still not quite right.

Before:
image

After:
image

**Comment by beaugrantham-va** [comment]
This also changes the style of the non-refactored page. The login buttons are larger and the create-account button is misaligned. The old page needs to be unaffected.

image

**Comment by crolarlibertyva** [comment]
It was determined that a lighter touch refactor was a safer change for lighthouse.

**Pull request closed without merging**